vagrant/provisioning/roles/confluent-platform-install/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+# tasks file for confluent-install
+- name: Install curl and which
+  yum:
+    name: "{{ packages }}"
+  vars:
+    packages:
+    - curl
+    - which
+
+- name: Install Confluent Platform public key
+  rpm_key:
+    state: present
+    key: https://packages.confluent.io/rpm/5.4/archive.key
```

Comment:
Let's pull the version (5.4) into a fact, so we can upgrade or change version just by changing the facts file.